views/js/controller/list/index.js

```diff
@@ -258,58 +274,103 @@ define([
+                    <li id="list-element_0">
+                        <span class="list-element" id="list-element_0_${Uri.encode(newList.elements[0].uri)}">${newList.elements[0].label}</span>
+                    </li>
+                </ol>
+            </div>
+            <footer class="data-container-footer action-bar">
+                <button
+                    type="button"
+                    title="${__('Edit this list')}"
+                    class="icon-edit list-edit-btn btn-info small rgt"
+                    data-uri="${newList.uri}"
+                ></button>
+                <button
+                    type="button"
+                    title="${__('Delete this list')}"
+                    class="icon-bin list-delete-btn btn-warning small rgt"
+                    data-uri="${newList.uri}"
+                ></button>
+            </footer>`
+        );
+    }
+
+    function createRemoteListContainer(newList) {
+        let listElements = '';
+
+        newList.elements.forEach((element, index) => {
+            listElements += `<li id="list-element_${index}">
+                <span class="list-element" id="list-element_0_${Uri.encode(element.uri)}">${element.label}</span>
+            </li>`;
+        });
+
+        return $(
+            `<section id="list-data_${newList.uri}" class="data-container list-container">
+            <header class="container-title">
+                <h6>${newList.label}</h6>
+            </header>
+            <div class="container-content" id="list-elements_${newList.uri}">
+                <ol>${listElements}</ol>
+            </div>
+            <footer class="data-container-footer action-bar">
+                <button
+                    type="button"
+                    title="${__('Reload this list')}"
+                    class="icon-reload list-reload-btn btn-info small rgt"
+                    data-uri="${newList.uri}"
+                ></button>
+                <button
+                    type="button"
+                    title="${__('Delete this list')}"
+                    class="icon-bin list-delete-btn btn-warning small rgt"
+                    data-uri="${newList.uri}"
+                ></button>
+            </footer>
+        </section>`
+        );
+    }
 
     function getUriValue(targetUri) {
         if (typeof targetUri === 'string') {
             return targetUri;
         } else if (targetUri.currentTarget){
             return $(targetUri.currentTarget).data('uri');
         }
-    };
+    }
 
     return {
-
-        /**
-         * The list controller entrypoint
-         */
+        // The list controller entrypoint
         start() {
             $('.form-submitter').off('click').on('click', (function (e) {
                 e.preventDefault();
-                handleCreateList();
+
+                const $form = $(e.target).closest('form');
+
+                urlUtil.route('remote', 'Lists', 'taoBackOffice').includes($form.attr('action'))
```

Comment:
I would move it to a function to explain better what is checking.

Reply:
```suggestion
                $form.attr('action').includes('remote')
```

Reply:
checking that action url contains `remote` substring